### PR TITLE
fix suffixes for kk language

### PR DESCRIPTION
**Pull request by talgautb**
Fixed wrong suffixes for kazakh language, for example `29-ші мамыр 1453 ж.` should be `29-шы мамыр 1453 ж.`

p.s. if it's ok, can we add label `hacktoberfest-accepted` to PR ?

### CODE REVIEW

File: src/locale/kk/_lib/localize/index.js [hunk 1]
```diff
@@ -134,21 +134,34 @@ var formattingDayPeriodValues = {
...
-    suffix = '-ші'
-  } else {
-    suffix = '-ші'
-  }
-
-  return dirtyNumber + suffix
+function ordinalNumber(dirtyNumber) {
+  var number = Number(dirtyNumber)
+  var a = number % 10,
+    b = number >= 100 ? 100 : null
```
Comment (drugoi) [inline src/locale/kk/_lib/localize/index.js:163]: Declare a new variable, please. 
It's better to understand and read.

Comment (talgautb) [reply src/locale/kk/_lib/localize/index.js]: @drugoi done

**Review by drugoi — APPROVED** [review]

**Review by kossnocorp — APPROVED** [review]
Thank you a lot!

**Comment by kossnocorp** [comment]
@talgautb the PR will count as a Hacktoberfest one because we have the tag in the repo.

**Comment by drugoi** [comment]
Hi, @kossnocorp. In addition, please, add @talgautb to @date-fns/i18n-kk team, he is a Kazakh native speaker.

**Review by tan75 — APPROVED** [review]
Thank you.

**Pull request merged**